### DISCUSSION TITLE
show only the control bar

### DISCUSSION THREAD

**Discussion by ghost** [smplayer-dev/smplayer#17] [Ideas]
I am new to SMplayer, and am currently using version 19.10.2 as it is available in the Linux Mint 20.1 repository. 

I couldn't find a related topic via the search. Is there anyway to use SMplayer while only showing the control bar (that is, no menu bar, no main toolbar, and no status bar - just the control bar). I know there is a way to not show the main toolbar - but the mini and mpc GUI options do not seem to allow for only the control bar to be shown. I realize devs likely have more pressing things to focus on - I was simply hoping this might be an option at some point. I am really loving SMplayer with MPV multimedia engine. Thank you to the devs for this great software, and thank your for your time in reading this.

**Comment by smplayer-dev** [comment]
Enable compact mode (menu Video) and enable the option "Display in compact mode too" in preferences -> interface -> floating control. This the most similar to what you ask for.

**Reply by ghost** [reply]
It's precisely what I was looking for! Thanks again for all your work, and taking the time to answer my question. I will certainly be donating to you as soon as I can.

**Reply by Bhaalspawn** [reply]
> 
> 
> Enable compact mode (menu Video) and enable the option "Display in compact mode too" in preferences -> interface -> floating control. This the most similar to what you ask for.

Hello
This is working only with default interface (and the toolbar keep disappearing). it's not stable 
I know it's compact mode but if i enable "show also in compact mode", it should be always visible and not only when moving the mouse.
